### DISCUSSION TITLE
How to pass arguments to extracted EXE?

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5164] [Help and Questions] [answered]
How to pass arguments to extracted EXE? Or better said how to shim the specific exe with specific arguments and also create shortcut for it with arguments?

I know "shortcut" and "bin" commands and all that, but how to properly pass args?    For example i'd like to shim and 'shortcut' exe named "PCTray.exe", but it needs the argument / flag for it to work properly.. like  `PCTray.exe /showui`, in Windows Explorer that is represented as `"..\SomeExamplePath\PCTray.exe" /showui `  .  So how to pass that **/showui** flag,

Thanks

**Comment by r15ch13** [comment, marked as answer]
Like this? ([alacritty.json](https://github.com/ScoopInstaller/Extras/blob/master/bucket/alacritty.json))
```json
    "bin": [
        "alacritty.exe",
        [
            "alacritty.exe",
            "alacritty-config",
            "--config-file \"$persist_dir\\alacritty.yml\""
        ]
    ],
    "shortcuts": [
        [
            "alacritty.exe",
            "Alacritty",
            "--working-directory \"%USERPROFILE%\""
        ],
        [
            "alacritty.exe",
            "Alacritty with Configuration",
            "--config-file \"$persist_dir\\alacritty.yml\""
        ]
    ],
```


```json
    "bin": [
        "PCTray.exe",
        [
            "PCTray.exe",
            "PCTray",
            "/showui"
        ]
    ],
    "shortcuts": [
        [
            "PCTray.exe",
            "PCTray",
            "/showui"
        ]
    ],
```

**Reply by anderlli0053** [reply]
@r15ch13  Thank you very much!
